### ISSUE TITLE
How do I completely uninstall split-vpn/revert to defaults

### ISSUE THREAD

**Issue by diogosalazar** [peacey/split-vpn#63]
Hi, how can I completely undo/uninstall split-vpn from my system? Thanks.

**Comment by peacey** [comment]
Hi @diogosalazar,

Here is the process to uninstall the script completely:

1. Bring down any tunnel you are using with the script. 
2. Delete the script folder at `/mnt/data/split-vpn/vpn`. If you want to delete your configuration files and not just the script, then also delete `/mnt/data/split-vpn` completely. (Note if you are using a UDM-SE/UDR, the folder is /data/split-vpn).
3. Delete the boot script if you are using one.
    a. If using a UDM/UDM-Pro, the boot script is at located at`/mnt/data/on_boot.d/99-run-vpn.sh`.
    b. If using a UDM-SE/UDR, the boot service is located at `/etc/systemd/system/run-vpn.service`.
4. Reboot the system to make sure nothing starts on boot.

Basically, run these commands to delete everything including configuration then reboot:

```sh
rm -rf /mnt/data/split-vpn /data/split-vpn
rm -rf /mnt/data/on_boot.d/99-run-vpn.sh /etc/systemd/system/run-vpn.service
```

**Comment by diogosalazar** [comment]
Thank you @peacey!
